presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -341,11 +361,11 @@ void PeriodicTaskManager::updateCacheStats() {
         kCounterSsdCacheCumulativeWrittenBytes,
         memoryCacheStats.ssdStats->bytesWritten);
     RECORD_METRIC_VALUE(
-        kCounterSsdCacheCumulativeCachedEntries,
-        memoryCacheStats.ssdStats->entriesCached);
+        kCounterSsdCacheCumulativeAgedOutEntries,
+        memoryCacheStats.ssdStats->entriesAgedOut)
     RECORD_METRIC_VALUE(
-        kCounterSsdCacheCumulativeCachedBytes,
-        memoryCacheStats.ssdStats->bytesCached);
+        kCounterSsdCacheCumulativeAgedOutRegions,
+        memoryCacheStats.ssdStats->regionsAgedOut);
```

Comment:
I'm a bit confused with all these cumulative counters.
What is the value of them?
I would assume we are interested in the instantaneous number rather than 'from the start of the server'...

Reply:
Yes, the cumulative counters are "from the start of the server". And we can only collect the cumulative counters, primarily due to our current setup of counter reporting and collecting I think.